po/fr.po

```diff
@@ -4723,8 +4723,8 @@ msgstr "Envoyer par _mail"
 msgid ""
 "Sends the opened file as unzipped attachment by any mailer from your $PATH"
 msgstr ""
-"Envois le fichier ouvert comme pièce jointe non-compressée avec n'importe "
-"quel client."
+"Envoie le fichier ouvert comme pièce jointe non-compressée avec n'importe "
+"quel client depuis votre $PATH"
```

Comment:
I think I was the one leaving this part of the sentence because I feel it's heavy and really pointless, but yeah technically it's there.

--

Ceci dit, je driais plutôt "dans votre $PATH", par ce que la question ici est si le client mail est dans un dossier inclu dans la liste `$PATH`, `$PATH` n'est pas le conteneur lui-même.